test/reactions/github/pull-request-merged.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { PullRequestMerged } from '../../../src/reactions/github/pull-request-merged';
+import { TwitchChat } from '../../../src/services/TwitchChat';
+import { StreamLabs } from '../../../src/services/StreamLabs';
+
+describe('PullRequestMerged', () => {
+	describe('#handle', () => {
+		let payload: any;
+		let streamlabs: StreamLabs;
+		let twitchChat: TwitchChat;
+
+		beforeEach(() => {
+			payload = {
+				action: 'closed',
+				repository: {
+					full_name: 'streamdevs/webhook',
+					html_url: 'https://github.com/streamdevs/webhook',
+				},
+				pull_request: { user: { login: 'SantiMA10' }, merged: true },
+				sender: { login: 'pepe' },
+			};
+
+			twitchChat = ({
+				send: jest.fn(),
+			} as unknown) as TwitchChat;
+			streamlabs = ({
+				alert: jest.fn(),
+			} as unknown) as StreamLabs;
+		});
+
+		it("returns 'twitchChat.notified' if something goes wrong with TwitchChat", async () => {
+			const twitchChat = ({
+				send: jest.fn(async () => {
+					throw new Error('boom');
+				}),
+			} as unknown) as TwitchChat;
+			const subject = new PullRequestMerged(twitchChat, streamlabs);
+
+			const {
+				twitchChat: { notified },
+			} = await subject.handle({ payload });
+
+			expect(notified).toEqual(false);
+		});
+
+		it("returns 'streamlabs.notified' if something goes wrong with StreamLabs", async () => {
```

Comment:
This message is not descriptive enough. 
Maybe "Sets streamlabs.notified = false when something goes wrong [...]" ?

Reply:
ups!